tests/test_converter.py

```diff
@@ -253,23 +254,60 @@ def test_extra_parameters_were_passed(self, recorder):
             with recorder:
                 quantum_circuit(params={theta: 0.5, phi: 0.3})
 
-    def test_crz(self, recorder):
-        """Tests loading a circuit with the controlled-Z operation."""
+    @pytest.mark.parametrize("qiskit_operation, pennylane_name", [(QuantumCircuit.crx, "CRX"), (QuantumCircuit.crz, "CRZ")])
+    def test_controlled_rotations(self, qiskit_operation, pennylane_name, recorder):
+        """Tests loading a circuit with two qubit controlled rotations (except
+        for CRY)."""
 
         q2 = QuantumRegister(2)
         qc = QuantumCircuit(q2)
-        qc.crz(0.5, q2[0], q2[1])
+
+        qiskit_operation(qc, 0.5, q2[0], q2[1])
+
 
         quantum_circuit = load(qc)
 
         with recorder:
             quantum_circuit()
 
         assert len(recorder.queue) == 1
-        assert recorder.queue[0].name == 'CRZ'
+        assert recorder.queue[0].name == pennylane_name
         assert recorder.queue[0].params == [0.5]
         assert recorder.queue[0].wires == [0, 1]
 
+    def test_cry(self, recorder):
+        """Tests that the decomposition of the controlled-Y operation is being
+        loaded."""
+        # This test will be merged into the test_controlled_rotations test once
+        # the native CRY is used in Qiskit and not a set of instructions
+        # yielded from decomposition is being added
```

Comment:
Oh, I see!

Reply:
How come this is the case? Is Qiskit doing the decomposition? Or PL core?

Reply:
It's coming from Qiskit, but the question of why it is so got me wondering as well :sweat_smile: :
```python
from qiskit import QuantumCircuit, QuantumRegister

q2 = QuantumRegister(2)
qc = QuantumCircuit(q2)

instr_set = qc.cry(0.5, q2[0], q2[1])
instr_set.data
```
Outputs:
```python
[(<qiskit.extensions.standard.u3.U3Gate object at 0x7fec26d88350>, [Qubit(QuantumRegister(2, 'q2'), 1)], []), (<qiskit.extensions.standard.x.CXGate object at 0x7fec26d88510>, [Qubit(QuantumRegister(2, 'q2'), 0), Qubit(QuantumRegister(2, 'q2'), 1)], []), (<qiskit.extensions.standard.u3.U3Gate object at 0x7fec26d885d0>, [Qubit(QuantumRegister(2, 'q2'), 1)], []), (<qiskit.extensions.standard.x.CXGate object at 0x7fec26d88610>, [Qubit(QuantumRegister(2, 'q2'), 0), Qubit(QuantumRegister(2, 'q2'), 1)], [])]
```